pkg/db/cloudsql.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import (
+	"fmt"
+	"strings"
+
+	_ "github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/dialers/postgres" // Blank import used for registering cloudsql driver as a database driver
+	"github.com/jmoiron/sqlx"
+)
+
+// newCloudSQL returns a new Google Cloud SQL database
+func newCloudSQL(cfg Config) (datab, error) {
+	return NewCloudPostgres(cfg)
+}
+
+// NewCloudPostgres returns a new Google Cloud Postgres database instance
+func NewCloudPostgres(cfg Config) (*Postgres, error) {
+	cfg.Path = strings.Trim(cfg.Path, " ")
+	cfg.Path = strings.Trim(cfg.Path, "'")
```

Comment:
As per to Go documentation for `Trim`: `all leading and trailing Unicode code points contained in cutset removed`

So you should be able to simplify this into one call by adding ` ` & `'` into the same string